### DISCUSSION TITLE
How to change the root page (hello page) to custom one

### DISCUSSION THREAD

**Discussion by bcastillov-esp** [pretix/pretix#2005] [Q&A] [answered]
Hello! Thanks for developing this project, we have managed to install it manually without errors and everything works really well.
But we have a doubt, we want to modify the root page that greets the visitor to pretix (the hello page), with a more personalized frontend page with our logo and that can give way to the list of public URLs of the organizers.

We have already developed this page, but we don't know where to place it in pretix to replace the default root page; Or if this is not possible, how to configure the nginx so that when the url is www.example.com it shows our root page and when the url is www.example.com/control or www.example.com/organizername or any other request_uri make the reserved proxy to localhost: 8345 and let pretix do its job.

Thanks for the help and the explanation that you can give us.

**Comment by raphaelm** [comment, marked as answer]
There is no feature for this in pretix itself. We recommend to do this in nginx, as you mentioned:

```
location = / {
    return 301 https://mywebsite.com;
}
```

**Reply by raphaelm** [reply]
I think this goes beyond what we can provide support here. You can't just send a POST request to a pretix page, that's why you get the error.

Why did the approach mentioned in my comment not work? Can you show the full nginx config?

**Reply by bcastillov-esp** [reply]
Hi Raphaelm

Since our last post we have made changes to the configuration and we can already say that it works as we wanted!

Our problem was the ignorance that we have in these linux environments and their applications to configure services on the internet. But nothing that cannot be solved thanks to your recommendation and a bit of google and youtube :)

We have created a subdomain and in it we have installed pretix, manually and working perfectly.
In the main domain we have put our frontend and with nginx we have created returns 301/302 to the URLs of our organizers that are in our pretix subdomain.

We have created a table of returns in nginx to be able to handle them more easily.
In our frontend page we perform the redirection invoking the location \ organizer of our main domain nginx configuration and the corresponding organizer page is already displayed without problems.

We have been experimenting with the option to use widgets, but although they are much easier to use, some functionalities were lost and we wanted to keep them.

So it works for us and we are very happy with the result. Thank you very much for all the help you have given us from the beginning, your patience with us and above all for allowing your development to be for public use, that is very much appreciated.

If you see something in the configuration that is wrong or can be done better, we are listening to learn from what you tell us.

Greetings and again thank you very much, we are going to experiment with the addons so we will surely ask you more things :)

[nginx_maindomain_conf_frontend.txt](https://github.com/pretix/pretix/files/6210866/nginx_maindomain_conf_frontend.txt)
[nginx_subdomain_conf_pretix.txt](https://github.com/pretix/pretix/files/6210868/nginx_subdomain_conf_pretix.txt)
